authentik/core/api/property_mappings.py

```diff
@@ -67,6 +74,18 @@ class Meta:
         ]
 
 
+class PropertyMappingFilterSet(FilterSet):
+    """Filter for PropertyMapping"""
+
+    managed = extend_schema_field(OpenApiTypes.STR)(AllValuesMultipleFilter(field_name="managed"))
+
+    managed__isnull = BooleanFilter(field_name="managed", lookup_expr="isnull")
```

Comment:
Kept this filter as-is (i.e. no changes to the API), but we can rename it if we want.